website/docs/manager_ftrack_actions.md

```diff
@@ -220,47 +194,106 @@ You should use this action if you need to delete Entities or Asset Versions othe
 
 *Create Project Structure* helps to create basic folder structure and may create the main ftrack entities for the project.
 
-Structure is loaded from [presets](admin_config#presets) *(Pype Settings → Project → Global → Project Folder Structure)*. You should examine that preset to see how it works. Preset may contain dictionaries of nested dictionaries where each key represents a folder name. Key and all it's parents will be also created in Ftrack if the key ends with `[ftrack]`. Default Ftrack entity type is *Folder* but entity type can be specified using `[ftrack.{entity type}]`. To create *Sequence* with name *Seq_001* key should look like `Seq_001[ftrack.Sequence]`.
+Structure is loaded from settings *(OpenPype Settings → Project → Global → Project Folder Structure)*. You should examine these settings to see how it works. Settings may contain dictionaries of nested dictionaries where each key represents a folder name. Key and all it's parents will be also created in Ftrack if the key ends with `[ftrack]`. Default Ftrack entity type is *Folder* but entity type can be specified using `[ftrack.{entity type}]`. To create *Sequence* with name *Seq_001* key should look like `Seq_001[ftrack.Sequence]`.
 
 :::note
 Please keep in mind this action is meant to make your project setup faster at the very beginning, but it does not create folders for each shot and asset. For creating asset folder refer to `Create Folders` Action
 :::
 
 ---
-<div class="ImagePanel" data-image-align='right'>
-<div class="ImagePanel.content">
+<div class="row markdown">
+<div class="col col--10 markdown">
+
+## Delivery
+
+</div>
+<div class="col col--2 markdown">
+
+![ftrack-delivery-icon](assets/ftrack/ftrack-delivery-icon.png)
+
+</div>
+</div>
+
+* Entity types: Task
+* User roles: Pypeclub, Project manager, Administrator
+
+Collects approved hires files and copy them into a folder. It usually creates h.264 files for preview and mov for editorial. All files are then copied according to predefined naming convention to a specific folder.  
```

Comment:
This doesn't create any review files. It just takes any components of any versions and copies and renames them correctly.